hadoop-ozone/dist/src/main/smoketest/omha/om-roles.robot

```diff
@@ -19,12 +19,37 @@ Resource            ../commonlib.robot
 Test Timeout        5 minutes
 Test Setup          Run Keyword if    '${SECURITY_ENABLED}' == 'true'    Kinit test user     testuser     testuser.keytab
 
+*** Keywords ***
+Parse non json output
+     [Arguments]                     ${output}
+     Should Match Regexp             ${output}                  [om (: LEADER|)]
+
+Parse json output
```

Comment:
Similarly: `Assert Leader Present in JSON` or `JSON Output Should Contain Leader`.

Reply:
Done